src/providers/tpm_provider/utils.rs

```diff
@@ -233,13 +234,13 @@ pub fn signature_data_to_bytes(data: SignatureData, key_attributes: Attributes)
 }
 
 pub fn parsec_to_tpm_signature(
-    data: Vec<u8>,
+    data: &Zeroizing<Vec<u8>>,
     key_attributes: Attributes,
     signature_alg: AsymmetricSignature,
 ) -> Result<Signature> {
     Ok(Signature {
         scheme: convert_asym_scheme_to_tpm(Algorithm::AsymmetricSignature(signature_alg))?,
-        signature: bytes_to_signature_data(data, key_attributes)?,
+        signature: bytes_to_signature_data(data.to_vec(), key_attributes)?,
```

Comment:
That is one of those place in the TPM crate where we need to check zeroing, or only take a reference if possible.

Reply:
We can actually send the `Zeroizing` down in `bytes_to_signature_data` where we convert to the TSS type, no need to clone the buffers.

Reply:
Alternatively we can use `AsMut<[u8]>`

Reply:
Yeah, this one I'll change `bytes_to_signature_data` to take a `Zeroizing`